.gitpod.Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM gitpod/workspace-full
 
-RUN bash -cl "rustup toolchain install stable && rustup target add wasm32-unknown-unknown"
+ENV CARGO_HOME=/home/gitpod/.cargo
+
+RUN bash -cl "rustup default stable-2020-10-08-x86_64-unknown-linux-gnu && rustup target add wasm32-unknown-unknown"
```

Comment:
it looks like we default to `stable-2020-10-08` since we specify it in the `rust-toolchain` file. Should we upgrade it to latest stable? But otherwise everything else looks fine

Reply:
Great comments, thanks! Pushed up some changes